src/test/resources/com/dnsimple/deleteZoneRecord/success_non_empty.http

```diff
@@ -0,0 +1,18 @@
+HTTP/1.1 204 No Content
+Server: nginx
+X-Request-Id: 1740e15a-179f-4e02-a47e-7414339b2be3
+X-Content-Type-Options: nosniff
+X-Runtime: 0.105620
+Connection: keep-alive
+X-Download-Options: noopen
+X-RateLimit-Reset: 1589359725
+X-Permitted-Cross-Domain-Policies: none
+Date: Wed, 13 May 2020 08:11:42 GMT
+X-Frame-Options: DENY
+Strict-Transport-Security: max-age=31536000
+X-RateLimit-Remaining: 4786
+Cache-Control: no-cache
+X-RateLimit-Limit: 4800
+X-XSS-Protection: 1; mode=block
+
+ 
```

Comment:
Note that there's actually a whitespace here

Reply:
@ggalmazor please help me to understand. Is this response returned from our API?

Reply:
No. It's created by me. I've been unable to actually observe a non-empty 204 from our API (I've tried debugging the HTTP client and using `curl`), but our API client behaves as if the 204 response wasn't empty. 

I can't be sure if curl or the HTTP client lib we're using are trimming it before showing it to me, though.

In any case, this `.http` file will reproduce the exact error stacktrace we're getting when using the API client for real, so it's enough to fix the code for now.